src/interface/storage.thrift

```diff
@@ -203,6 +205,40 @@ struct CatchUpDataReq {
     3: common.HostAddr     target,
 }
 
+struct UpdateResponse {
+    1: required ResponseCommon result,
+    2: optional common.Schema schema,   // return column related props schema
+    3: optional binary data,            // return column related props value
+    4: optional bool upsert = false,    // it's true when need to be inserted by UPSERT
```

Comment:
What' the purpose about this field in response

Reply:
If insert the edge in UPSERT sentence, the response should be notified, then send the request for inserting the reversely edge.

Reply:
make sense